charts/addons/values.yaml

```diff
@@ -1,66 +1,74 @@
 ---
-# Configure Traefik for load balancing:
-#   If enabled: true, Traefik is deployed automatically.
-#   If enabled: false, Traefik is not deployed and load balancing must be configured manually.
-#   Pega recommends enabling Traefik on providers other than Openshift and eks.
-#   On Openshift, Traefik is ignored and Pega uses Openshift's built-in load balancer.
-#   On eks it is recommended to use aws alb ingress controller.
+# Load balancer parameters
+# Pega deployments support the use of Traefik as the default load balancer; however, by default, 
+# Pega Platform deployments assume clients will use the load balancing tools featured in the
+# Kubernetes environment of the deployment. For each environment the default load balancer is listed:
+# Openshift: Openshift built-in load balancer.
+# EKS: AWS alb ingress controller, which works without further configuration.
+# AKS: Application Gateway - configure an Ingress Controller (AGIC) in AKS to use it.
+# GKE: Google Cloud Load Balancer (GCLB), which works without further configuration.
+# Pega recommends enabling Traefik for PKS and k8s provider.
+# To override the use of the default load balancer in AKS or GKE, you enable Traefik by setting the
+# parameter traefik.enabled: true.
+# When traefik.enabled: false (the default) the deployment ignores the remaining Traefik parameters and 
+# the deployment uses the load balancer tools featured Kubernetes environment of the deployment.
 traefik:
-  enabled: true
-  # Set any additional Traefik parameters. These values will be used by Traefik's helm chart.
+  enabled: false
+  # When you set this to true, you can then set additional Traefik parameters to be passed into Traefik's Helm chart.
   # See https://github.com/helm/charts/blob/master/stable/traefik/values.yaml
-  # Set traefik.serviceType to "LoadBalancer" on gke, aks, and pks
-  serviceType: NodePort
-  # If enabled is set to "true", ssl will be enabled for traefik
+  # To use Traefik as a load balancer in PKS, AKS, GKE, or k8s, set traefik.serviceType: "LoadBalancer".
+  serviceType: LoadBalancer
```

Comment:
Should this be `NodePort` by default?  The comment above says to _change_ it to `LoadBalancer` iff using specified cloud providers.